Src/DependencyCollector/DependencyCollector/Microsoft.ApplicationInsights.DependencyCollector.targets

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="CopyApplicationInsightsConfigToOutputDirectory" AfterTargets="CopyFilesToOutputDirectory" >
```

Comment:
can you please set affected files as a property of this target

Reply:
sorry, i don't understand what you're asking here